services/reventService.tsx

```diff
@@ -238,3 +238,106 @@ export async function getContestList() {
   return contests
   //   return fetchEndpoint('contests/', GET)
 }
+
+export async function getNotificationsList() {
```

Comment:
you should type the return type/interface for your services, that way you make sure that it's returning what you're expecting

Reply:
in this case a list of the `Notification` interface